core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraIdentityProvider.java

```diff
@@ -69,7 +69,7 @@
 public class TestingHydraIdentityProvider
         implements AutoCloseable
 {
-    private static final String HYDRA_IMAGE = "oryd/hydra:v1.10.6";
+    private static final String HYDRA_IMAGE = "oryd/hydra:v1.11.9";
```

Comment:
nit: Do we have any release notes for the new version, which we could link as a part of the PR description.